enterprise/admin/forms.py

```diff
@@ -522,6 +523,20 @@ def clean(self):
                 ),
             )
 
+        # There can be only default provider in case an enterprise has multiple providers.
+        if default_provider is True:
+            default_provider_forms = int(self.data.get('enterprise_customer_identity_providers-TOTAL_FORMS'))
+            default_provider_values = [self.data.get('enterprise_customer_identity_providers-' +
```

Comment:
as list comprehension has been spread to multiple lines should not we use simple for loop?

Reply:
In that case, we need to create an empty list and then append the data in that. I think it looks cleaner than that solution.